pymongo/asynchronous/cursor.py

```diff
@@ -1275,6 +1288,9 @@ async def __aexit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
     async def to_list(self) -> list[_DocumentType]:
         return [x async for x in self]  # noqa: C416,RUF100
 
+    async def to_list_batch(self) -> list[_DocumentType]:
```

Comment:
What is to_list_batch, was it only added for testing purposes?

Reply:
Forgot to rename it, fixed.